openslides/motions/config_variables.py

```diff
@@ -118,11 +116,19 @@ def get_config_variables():
         name='motions_amendments_prefix',
         default_value='A',
```

Comment:
Translation the English `A` into German `Ä` on server side seams to be not trivial.

Suggestion: Change default config value to something untranslatable like `–` or `-` or `~` or `>`. Every admin use may change the value to his preference at any time in the config.